src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -321,7 +321,10 @@ public long MaxRequestContentBufferSize
 
         public int MaxResponseHeadersLength
         {
-            get { return 0; } // TODO: Issue #8541 - return default value used in Wininet.
+            // Windows.Web.Http is built on WinINet. There is no maximum limit (except for out of memory)
+            // for received response headers. So, returning 0 (indicating no limit) is appropriate.
+            get { return 0; }
```

Comment:
And further, the default value for .NET Framework `HttpClientHandler.MaxRequestContentBufferSize` is actually this:

```c#
        internal const long MaxBufferSize = Int32.MaxValue;
```

In practice, Int32.MaxValue is basically "unlimited", i.e. same as actually setting 0.

@stephentoub Do you suggestions on how we should converge for "unlimited"?